app/src/models/UserFile.ts

```diff
@@ -100,6 +106,14 @@ export class UserFile
           type: DataTypes.STRING(255),
           allowNull: true,
         },
+        subsectors: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+        },
+        scopes: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+        },
```

Comment:
These should not be CSV strings. Use string/ number arrays or, even better, use hasMany relationships here. This would make it much easier to find the files belonging to a specific subsector or scope in the database.